Library/Formula/gjs.rb

```diff
@@ -0,0 +1,50 @@
+class Gjs < Formula
+  desc "Javascript Bindings for GNOME"
+  homepage "https://wiki.gnome.org/Projects/Gjs"
+  url "https://download.gnome.org/sources/gjs/1.44/gjs-1.44.0.tar.xz"
+  sha256 "88c960f6ad47a6931d123f5d6317d13704f58572f68a4391913a254ff27dce80"
+
+  depends_on "pkg-config" => :build
+  depends_on "gobject-introspection"
+  depends_on "nspr"
+  depends_on "readline"
+
+  resource "mozjs24" do
+    url "http://ftp.mozilla.org/pub/mozilla.org/js/mozjs-24.2.0.tar.bz2"
+    sha256 "e62f3f331ddd90df1e238c09d61a505c516fe9fd8c5c95336611d191d18437d8"
+  end
+
+  def install
+    resource("mozjs24").stage do
+      cd("js/src") do
+        # patches taken from MacPorts
+        inreplace "config/milestone.pl", "if (defined(@TEMPLATE_FILE)) {", "if (@TEMPLATE_FILE) {"
+        inreplace "config/rules.mk", "@executable_path", "${prefix}/lib"
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!